CLAUDE.md

```
@@ -6,72 +6,223 @@ Instructions for Claude Code when working with this repository.
+- **Server**: `kubectl config view -o jsonpath='{.clusters[0].cluster.server}'`
+- **LoadBalancer IP**: `kubectl get svc -n ingress-nginx ingress-nginx -o jsonpath='{.status.loadBalancer.ingress[0].ip}'`
+- **Storage Class**: microk8s-hostpath
 
 ## Key Information
 
-### Authentik Credentials
-- Admin: akadmin / Admin123!
-- Database password: WNAkt8ZouZRhvlcf3HSAxFXQfbt4qszs
-- No ArgoCD management (prevents password conflicts)
+### Access
+- **Authentik**: http://auth.local.test
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)